packages/engine/Source/Scene/GltfLoader.js

```diff
@@ -1593,17 +1616,20 @@ function loadMaterial(loader, gltfMaterial, frameState) {
       pbrSpecularGlossiness,
       frameState
     );
-  } else {
+  } else if (material.unlit === false) {
     if (defined(pbrMetallicRoughness)) {
       material.metallicRoughness = loadMetallicRoughness(
         loader,
         pbrMetallicRoughness,
         frameState
       );
     }
-    if (defined(pbrSpecular)) {
+    if (defined(pbrSpecular && material.unlit === false)) {
```

Comment:
```suggestion
    if (defined(pbrSpecular) && !material.unlit) {
```

I assume the second parenthesis for `defined` is in the wrong position. Secondly, can we simply like in my comment above?

Reply:
Thanks, good catch